docs/model-dev-guide/model-management/checkpoints.rst

```diff
@@ -40,69 +40,61 @@ ascending order.
-:meth:`~determined.experimental.client.Experiment.top_n_checkpoints` method. Only the single best
-checkpoint from each trial is considered; out of those, the checkpoints with the best validation
-metric values are returned in sorted order, with the best one first. For example, the following
-snippet returns the top five checkpoints from distinct trials of a specified experiment.
+You may also query checkpoints by preset checkpoint parameters, as defined by
+:class:`~determined.experimental.checkpoint.CheckpointSortBy`. The following code snippet fetches
+all checkpoints for the experiment sorted by the checkpoint's trial ID in descending order.
 
 .. code:: python
 
-   from determined.experimental import client
-
-   checkpoints = client.get_experiment(id).top_n_checkpoints(5)
+   from determined.experimental import checkpoint, client
 
-This method also accepts ``sort_by`` and ``smaller_is_better`` arguments.
+   checkpoints = client.get_experiment(id).list_checkpoints(
+       sort_by=checkpoint.CheckpointSortBy.TRIAL_ID,
+       order_by=checkpoint.CheckpointOrderBy.DESC
+   )
 
 :class:`~determined.experimental.client.Trial` is used for fine-grained control over checkpoint
-selection within a trial. It contains a :meth:`~determined.experimental.client.Trial.top_checkpoint`
-method, which mirrors :meth:`~determined.experimental.client.Experiment.top_checkpoint` for an
-experiment. It also contains :meth:`~determined.experimental.client.Trial.select_checkpoint`, which
-offers three ways to query checkpoints:
-
-#. ``best``: Returns the best checkpoint based on validation metrics as discussed above. When using
-   ``best``, ``smaller_is_better`` and ``sort_by`` are also accepted.
-#. ``latest``: Returns the most recent checkpoint for the trial.
-#. ``uuid``: Returns the checkpoint with the specified UUID.
+selection within a trial. It contains a
+:meth:`~determined.experimental.client.Trial.list_checkpoints` method, which mirrors
+:meth:`~determined.experimental.client.Experiment.list_checkpoints` for an experiment.
 
-The following snippet showcases how to use the different modes for selecting checkpoints.
+The snippet below demonstrates how to select the most recent checkpoint for a trial.
 
 .. code:: python
 
-   from determined.experimental import client
+   from determined.experimental import checkpoint, client
 
    trial = client.get_trial(id)
 
-   best_checkpoint = trial.top_checkpoint()
-
-   most_accurate_checkpoint = trial.select_checkpoint(
-       best=True, sort_by="accuracy", smaller_is_better=False
-   )
-
-   most_recent_checkpoint = trial.select_checkpoint(latest=True)
-
-   specific_checkpoint = client.get_checkpoint(uuid="uuid-for-checkpoint")
+   latest_checkpoint = trial.list_checkpoints(
+       sort_by=checkpoint.CheckpointSortBy.END_TIME,
+       order_by=checkpoint.CheckpointOrderBy.DESC,
+       limit=1
+   )[0]
 
 ********************************
  Using the ``Checkpoint`` Class
```

Comment:
There are a few more pointers to `top_checkpoint` in this file that, so long as you're fixing "Querying Checkpoints" should be replaced.

Reply:
Could you please search for other instances of the deprecated methods? `grep` turns up a couple for me for `top_checkpoint`: (`model-dev-guide/model-management/model-registry-org.rst` and `reference/python-sdk.rst`)